src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -239,8 +239,7 @@ private static String getFirstSentence(DetailNode ast) {
                 text = getFirstSentence(child);
             }
 
-            if (child.getType() != JavadocTokenTypes.JAVADOC_INLINE_TAG
-                && text.contains(periodSuffix)) {
+            if (text.contains(periodSuffix)) {
```

Comment:
Inline tags don't contain a text only it's children do, so this check was unnecessary.

Reply:
@rnveach , SummaryJavadoc is ANTLR based and part of google style, please pay attention to regression report for this Check.

Reply:
The javadoc:
`/** {@code test} */`
produces:
````
    |  |  |--BLOCK_COMMENT_BEGIN -> /* [2:0]
    |  |  |  |--COMMENT_CONTENT -> * {@code test}  [2:2]
    |  |  |  |  `--JAVADOC -> JAVADOC [2:3]
    |  |  |  |      |--TEXT ->  [2:3]
    |  |  |  |      |--JAVADOC_INLINE_TAG -> JAVADOC_INLINE_TAG [2:4]
    |  |  |  |      |  |--JAVADOC_INLINE_TAG_START -> { [2:4]
    |  |  |  |      |  |--CODE_LITERAL -> @code [2:5]
    |  |  |  |      |  |--WS ->  [2:10]
    |  |  |  |      |  |--TEXT -> test [2:11]
    |  |  |  |      |  `--JAVADOC_INLINE_TAG_END -> } [2:16]
    |  |  |  |      |--TEXT ->  [2:17]
    |  |  |  |      `--EOF -> <EOF> [2:18]
    |  |  |  `--BLOCK_COMMENT_END -> */ [2:16]
````

As shown, `JAVADOC_INLINE_TAG`'s text is just `JAVADOC_INLINE_TAG` and not the text `test`.